### DISCUSSION TITLE
Reference on creating custom renderer

### DISCUSSION THREAD

**Discussion by mdovn** [solidjs/solid#696] [Ideas]
Say I would like to combine pixijs with solid. Could you recommend me a good starting point?
I'm going to create something like https://reactpixi.org/, but for solid

**Comment by lxsmnsyc** [comment]
Not sure if this is helpful but the best reference so far would be Ryan's demo on custom renderers: https://www.youtube.com/watch?v=Yi_MJ8cVCCs

**Reply by mdovn** [reply]
Thanks, I'll take a look

**Comment by bobaekang** [comment]
You might have already found these, but there's [a README file](https://github.com/solidjs/solid/tree/main/packages/solid/universal#readme) for universal renderer module. And of course, [the release note for v1.2,0](https://github.com/solidjs/solid/releases/tag/v1.2.0) has a section on universal renderer. [The codesandbox example](https://codesandbox.io/s/custom-dom-renderer-yu2pj) mentioned there can also help--I believe this is more or less what Ryan put together during the livestreamed demo.

That said, I really hope that the post-v1.0 features get their proper treatment in official docs--sooner than later. Docs matter in adoption and these new big features are truly amazing ones that can put Solid ahead of other frameworks! I've expressed this sentiment at https://github.com/solidjs/solid/discussions/697.

**Reply by mdovn** [reply]
Thank you, your comment looks very useful to me
